src/main/scala/intent/sbt/Runner.scala

```diff
@@ -35,8 +35,8 @@ object IntentFingerprint extends SubclassFingerprint with
  * A single run of a single test (controlled by SBT)
  */
 class SbtRunner(
-  val args: Array[String],
-  val remoteArgs: Array[String],
+  val args: Array[String | Null],
```

Comment:
**Overridden java method returns array**
Overridden java method returns an array. Java arrays have type `Array[T | Null]`. Returned array must thereby be converted to have type `Array[T | Null]`.